controllers/helmchart_controller_test.go

```diff
@@ -694,6 +694,40 @@ var _ = Describe("HelmChartReconciler", func() {
 					storage.ArtifactExist(*got.Status.Artifact)
 			}, timeout, interval).Should(BeTrue())
 
+			When("Setting valid valuesFiles attribute", func() {
+				updated := &sourcev1.HelmChart{}
+				Expect(k8sClient.Get(context.Background(), key, updated)).To(Succeed())
+				chart.Spec.ValuesFiles = []string{
+					"./charts/helmchart/values.yaml",
```

Comment:
Note that these paths in the context of a `Bucket` or `GitRepository` resourced chart are relative to the respective source's root.

To be able to use these files you would need to commit them to the git repository as done [here](https://github.com/fluxcd/source-controller/blob/main/controllers/helmchart_controller_test.go#L463-L492). I hope that makes sense.

Reply:
It seems that the git repository in the tests uses this as the root:

```
A  testdata/charts/helmchartwithdeps/override.yaml
A  testdata/charts/helmchartwithdeps/values.yaml
```

Regardless, this doesn't solve the issue. The suite seems non-functional. I am going to focus on fixing the tests in #332.

Reply:
I assume this output above is from `git status -s`, the preceding `A` means the files were added but were not committed.